.travis.yml

```diff
@@ -4,19 +4,27 @@ python:
   - 3.3
   - 3.4
   - 3.5
-#  - 3.6
+  - 3.6
 env:
   - DJANGO=1.7
   - DJANGO=1.8
   - DJANGO=1.9
-#  - DJANGO=1.10
-#  - DJANGO=1.11
+  - DJANGO=1.10
+  - DJANGO=1.11
 matrix:
   exclude:
+    # Django 1.7 does not support Python 3.5+
     - python: 3.5
       env: DJANGO=1.7
+    - python: 3.6
+      env: DJANGO=1.7
+    # Django 1.9+ does not support Python 3.3
     - python: 3.3
       env: DJANGO=1.9
+    - python: 3.3
+      env: DJANGO=1.10
+    - python: 3.3
+      env: DJANGO=1.11
   include:
```

Comment:
Please, also add the exceptions for Python 3.6. I think the whole list should look like this:
```yml
matrix:
  exclude:
    - python: 3.5
      env: DJANGO=1.7
    - python: 3.6
      env: DJANGO=1.7
    - python: 3.6
      env: DJANGO=1.8
    - python: 3.3
      env: DJANGO=1.9
    - python: 3.6
      env: DJANGO=1.9
    - python: 3.3
      env: DJANGO=1.10
    - python: 3.6
      env: DJANGO=1.10
    - python: 3.3
      env: DJANGO=1.11
```

Reply:
I'm not sure I understand why Python 3.6 should be excluded: it works in production, and all the tests pass. Surely it's useful to test on the latest Pythons for the users that use it (myself included)?

Reply:
> I'm not sure I understand why Python 3.6 should be excluded

It's not our fault. Django versions below 1.11 don't officially support 3.6, that's it.

You can [browse throught the Django releases](https://pypi.python.org/pypi/Django) on PyPI and look at the meta data (trove classifiers). Python 3.6 is supported since [Django 1.11](https://pypi.python.org/pypi/Django/1.11).

Reply:
Right, I know they're not officially supported; but neither are Python 3.3 or Django 1.7 & 1.9, and the tests still cover those, because it's still useful for the people who use them.

Python is generally expected to be backward compatible, so it should be very reasonable to always test on all forward versions of Python that are known to work, and only exclude the ones that are known to be broken, and won't be fixed (like Django 1.7 and Python 3.5+).